genshi/compat.py

```diff
@@ -106,15 +126,13 @@ def build_code_chunk(code, filename, name, lineno):
                 co_name=name,
                 co_firstlineno=lineno,
             )
-        params =  [0, code.co_nlocals, code.co_kwonlyargcount,
-                  code.co_stacksize, code.co_flags | 0x0040,
-                  code.co_code, code.co_consts, code.co_names,
-                  code.co_varnames, filename, name, lineno,
-                  code.co_lnotab, (), ()]
-        if hasattr(code, "co_posonlyargcount"):
-            # PEP 570 added "positional only arguments"
-            params.insert(2, code.co_posonlyargcount)
```

Comment:
This attribute is never present here, so I removed the branch and cleaned things up a bit.